app/models/gql/gql.rb

```diff
@@ -285,14 +285,21 @@ def calculate_graphs
 
     def update_present
       instrument('gql.performance.present.update_present') do
-        scenario.inputs_present.each { |input, value| update_graph(present, input, value) }
```

Comment:
I'm nitpicking (again) and it doesn't matter now, but can I suggest that in the future we try to keep code-style/whitespace changes out of pull requests for new features? (same for [topology.rb](https://github.com/quintel/etengine/pull/963/files#diff-547f837cee3bb15737d186fdd290d314))

There style changes are A Good Thing™, but they're not related to deprecating initializer inputs.